orchagent/qosorch.cpp

```diff
@@ -647,28 +646,31 @@ task_process_status QosOrch::handleSchedulerTable(Consumer& consumer)
                 SWSS_LOG_ERROR("Unknown field:%s", fvField(*i).c_str());
                 return task_process_status::task_invalid_entry;
             }
-            if (SAI_NULL_OBJECT_ID != sai_object)
+        }
+
+        if (SAI_NULL_OBJECT_ID != sai_object)
```

Comment:
in line 601, we already check if it is null, when it is null, we return directly.